mw-webapp/src/utils/ThemeWorker.ts

```diff
@@ -106,6 +106,13 @@ const themedVariables: Record<string, Record<Theme, string>> = {
     // [Theme.DARK]: "rgb(152, 210, 235)",
     // [Theme.LIGHT]: "rgb(84,103,143)",
   },
+  activeBackgroundColor: {
```

Comment:
no such variable in design system. Discuss it with the designers

https://www.figma.com/file/ukyPUFcecdKzeBcdTHChYk/%D0%9F%D1%80%D0%BE%D0%B5%D0%BA%D1%82%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5?type=design&node-id=1924-38420&mode=design&t=h235YfXjqBolDCMl-0

Reply:
Victoria was on board with adding this colord. All right, I'll add some from the ones we can use